### PR TITLE
[GF] Sonar fixes in recently touched code

### LINKED ISSUE #6789: GoNLAnnotator has field postFixResource that should be a variable
SONAR showed it as a vulnerability that the field was public but the actual issue is that there is no need to make that variable a field.

### PR DESCRIPTION
**Pull request by fdlk**
Fix #6789: GoNLAnnotator has field postFixResource that should be a variable
Reduce complexity of GoNLAnnotator code.

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits

### CODE REVIEW

File: molgenis-api-tests/pom.xml [hunk 1]
```diff
@@ -51,6 +51,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>${awaitility.version}</version>
```
Comment (dennishendriksen) [inline molgenis-api-tests/pom.xml]: Remove version tag and add dependency to dependency management section in parent pom.xml to assert that the correct dependency is used for all tests.

File: molgenis-api-tests/pom.xml [hunk 2]
```diff
@@ -68,5 +74,11 @@
             <version>2.2.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```
Comment (dennishendriksen) [inline molgenis-api-tests/pom.xml]: Remove, duplicate dependency.

File: molgenis-api-tests/src/test/java/org/molgenis/api/tests/oneclickimporter/OneClickImporterControllerAPIIT.java [hunk 3]
```diff
@@ -132,19 +133,9 @@ private void oneClickImportTest(String fileToImport) throws URISyntaxException
...
-		while ((!jobStatus.equals("SUCCESS")) && pollIndex < maxPolls)
-		{
-			LOG.info("Import job status : " + jobStatus);
-			jobStatus = pollJobForStatus(jobUrl);
-			waitForNMillis(pollInterval);
-			pollIndex++;
-		}
-		LOG.info("Import job status : " + jobStatus);
-		assertEquals(jobStatus, "SUCCESS");
+		await().pollDelay(500, MILLISECONDS)
```
Comment (dennishendriksen) [inline molgenis-api-tests/src/test/java/org/molgenis/api/tests/oneclickimporter/OneClickImporterControllerAPIIT.java]: Could you write a section in the testing chapter of our developer guidelines containing an example like this?

File: molgenis-api-tests/src/test/java/org/molgenis/api/tests/utils/RestTestUtils.java [hunk 4]
```diff
@@ -122,43 +126,37 @@ public static void uploadEMX(String adminToken, String fileName)
...
 
-		String importJobURL = given().multiPart(file)
-									 .param("file")
-									 .param("action", "ADD_UPDATE_EXISTING")
-									 .header(X_MOLGENIS_TOKEN, adminToken)
-									 .post("plugin/importwizard/importFile")
-									 .then()
-									 .extract()
-									 .asString();
+		String importJobURLString = given().multiPart(file)
```
Comment (dennishendriksen) [inline molgenis-api-tests/src/test/java/org/molgenis/api/tests/utils/RestTestUtils.java]: Rename to stick to convention: importJobUrlString --> importJobURLString

File: molgenis-api-tests/src/test/java/org/molgenis/api/tests/utils/RestTestUtils.java [hunk 5]
```diff
@@ -122,43 +126,37 @@ public static void uploadEMX(String adminToken, String fileName)
...
-				sleep(500L);
-			}
-			catch (InterruptedException e)
-			{
-				LOG.error(e.getMessage());
-			}
-			LOG.info("Status: " + importStatus);
-		}
+		LOG.info("############ " + importJobURL);
+		Awaitility.await()
```
Comment (dennishendriksen) [inline molgenis-api-tests/src/test/java/org/molgenis/api/tests/utils/RestTestUtils.java]: Consistency: static import Awaitility

File: molgenis-core-ui/src/main/java/org/molgenis/data/importer/wizard/PackageWizardPage.java [hunk 6]
```diff
@@ -88,11 +92,11 @@ public String handleRequest(HttpServletRequest request, BindingResult result, Wi
 					// The entities that can not be imported. If even one entity can not be imported, everything fails
 					List<String> entitiesNotImportable = entitiesImportable.entrySet()
 																		   .stream()
-																		   .filter(entity -> entity.getValue() == false)
+																		   .filter(entity -> !entity.getValue())
 																		   .map(Map.Entry::getKey)
 																		   .collect(toList());
 
-					if (!entitiesNotImportable.isEmpty()) throw new RuntimeException(
+					if (!entitiesNotImportable.isEmpty()) throw new MolgenisDataException(
```
Comment (dennishendriksen) [inline molgenis-core-ui/src/main/java/org/molgenis/data/importer/wizard/PackageWizardPage.java]: Convention: use curly braces for single-line if statements

File: pom.xml [hunk 7]
```diff
@@ -82,7 +82,7 @@
         <testng.version>6.8</testng.version>
         <mockito.version>2.7.22</mockito.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
-
+        <awaitility.version>3.0.0</awaitility.version>
```
Comment (dennishendriksen) [inline pom.xml]: See previous comment about awaitility

File: molgenis-api-tests/src/test/java/org/molgenis/api/tests/oneclickimporter/OneClickImporterControllerAPIIT.java (hunk 3 above)
Comment (fdlk) [reply molgenis-api-tests/src/test/java/org/molgenis/api/tests/oneclickimporter/OneClickImporterControllerAPIIT.java]: Done!

**Pull request merged**